scripts/board.gd

```diff
@@ -149,6 +163,7 @@ func _draw_board():
 		if tile_type != "floor":
 			continue
```

Comment:
Just noticed that this behaviour is wrong. Tiles should still be created even if tile_type == "void", but they shouldn't be added to the floor_coordinates dict.

The reason the tile_type even exists is so we can assign textures to the void. Maybe something like a weird wormhole texture for a hole in the middle of the board.

Reply:
Doesn't need to be changed in this PR though.

Reply:
Added in bda5a8d - we only need to have a "void" texture...